lib/processors/lib/jaccard.py

```diff
@@ -0,0 +1,133 @@
+import random
+
+def jaccard_rec(current_answers):
```

Comment:
This is all Slaton's code. @slsp8752 Any chance you could take a look at Micha's line comments today?